src/helper/util.ts

```diff
@@ -251,6 +251,52 @@ export class ZkWasmUtil {
     return bns;
   }
 
+  static BNToHexString(bn: BN): string {
```

Comment:
Change the function name to start with lower case
 so Bytes to bytes, BN to bn

HexString to hexString